src/test/java/org/apache/sling/feature/cpconverter/handlers/UsersEntryHandlerTest.java

```diff
@@ -118,6 +115,18 @@ public void testUser() throws Exception {
         verify(aclManager, never()).addSystemUser(any(SystemUser.class));
     }
 
+    @Test
+    public void parseUserWithConfig() throws Exception {
+        String path = "/jcr_root/system/users/a/author/.content.xml";
```

Comment:
see above in the group test.... same applies here

Reply:
see above